File: zip_arrays/solutions/zip_arrays.cs
using System;
using System.Collections.Generic;
class MainClass {
  public static void Main (string[] args) {
    int[] array_one = {1, 5, 2};
    int[] array_two = {3, 6, 4};
    int[] zipped_array = new int[array_one.Length + array_two.Length];
    int count = 0;
    for (int i = 0; i < zipped_array.Length; i+=2) {
      zipped_array[i] = array_one[count];
      zipped_array[i + 1] = array_two[count];
      count += 1;
    }
    
    foreach(int element in zipped_array) {
      Console.WriteLine(element);
    }
  }
}

File: zip_arrays/solutions/zip_arrays.js
let array_one = [1, 5, 2];
let array_two = [3, 6, 4];
let zipped_array = [];

for (let i = 0; i < array_one.length; i++) {
  zipped_array.push(array_one[i]);
  zipped_array.push(array_two[i]);
}

console.log(zipped_array);